docs/signing-spec.md

```diff
@@ -242,14 +257,90 @@ by = "GitHub Action [https://github.com/exammple.com/hello_repo]"
 signature = "3dd237895ac..."
 key = "3c44..."
 role = "proxy"
+at = 1611960347
 
 # This identity asserts that it has hosted the bindle
 [[signature]]
 by = "Bindle Server [https://bindle.example.com]"
 signature = "2dd237895ac..."
 key = "2c44..."
 role = "host"
+at = 1611960357
 ```
+ Because of the timestamps, we can reconstruct the timeline and see that the key was
+ first signed by the creator, then signed by the proxy, and finally signed by the host.
+
+## Strategies of Key Trust
+
+This section is non-normative, describing how implementations MAY choose to behave.
+
+A Bindle may be signed by any number of keys in any number of roles.
+During verification, which of these keys must be verified?
+
+The simplest configuration of Bindle would require that the `creator` key be present in the keyring, and the content is then verified against this key's signature. All other keys MAY be verified.
```

Comment:
This states that all other keys may be verified.

Would it make sense to suggest that, if present, all other keys `SHOULD` be verified?

Reply:
Got to the next section where this is recommended.
Feel free to ignore this comment.